packages/contracts/test/voting/token-voting.ts

```diff
@@ -463,6 +489,23 @@ describe('TokenVoting', function () {
   });
 
   describe('Proposal + Execute:', async () => {
+    beforeEach(async () => {
+      const receivers = Array.from(Array(12).keys()).map(
+        i => signers[i].address
+      );
+      const amounts = Array(9).fill(10).concat([5, 4, 1]);
+
+      const balances = receivers.map((receiver, i) => {
+        return {
+          receiver: receiver,
+          amount: amounts[i],
+        };
+      });
+
+      await setBalances(balances);
+      await setTotalSupply(100);
```

Comment:
totalSupply is already 100, isn't it?
9 * 10 + 5 + 4 + 1 = 100

Reply:
I wanted to write it explicitly.
Yes, `setTotalSupply` sets the balance of `address(1)` to `BigNumber.from(totalSupply).sub(currentTotalSupply)` so it is not increased in this case. However, I wanted to explicitly set the value to 100 to be sure that this is the total supply.
If I set it to `setTotalSupply(99)`, the tests would fail `Error: value out-of-bounds` which is good.

The mocking functionality should be improved / replaced in a later task. It is not good, admittedly.

Reply:
I can also write an expect statement